ProjectSettings/EditorBuildSettings.asset

```diff
@@ -5,9 +5,54 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
+  - enabled: 1
+    path: Assets/MRTK/Examples/Experimental/ExamplesHub/Scenes/MRTKExamplesHub.unity
```

Comment:
Hmm, I'm not sure if we want to make this change or not. I like the simplicity of the repo being set up for just the HandInteractionExamples scene, which makes build / deploy / test a bit simpler. I'm interested to see what others think though.

We could probably add a repo script that's basically "set up build settings for Examples Hub" if we want the easy ability to build the hub?

Reply:
@keveleigh Actually, this build setting is specifically for MRTKExamplesHub. For these build settings, we already had other scenes. Just adding this one missing scene.

Reply:
![2020-05-18 11_12_52-Build Settings](https://user-images.githubusercontent.com/13754172/82245843-8cd06380-98f8-11ea-90eb-6636cac5628d.png)

Reply:
You won't see this list in other scenes.

Reply:
Thirteen other scenes are being added in this PR as well: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7902/files#diff-8f3c83eb79d0b4aa8cf581a168df971c

Reply:
Since this is the root ProjectSettings/EditorBuildSettings.asset, it'll change the default included scenes for the repo's project. I think the scene system adds missing scenes to the build settings, which might be what we're seeing here?

Reply:
@keveleigh Originally, we had all example scenes except this 'MRTKExamplesHub.unity' which doesn't make sense. (You need to include it to build examples hub)
I just added it.

![2020-05-18 11_18_53-Build Settings](https://user-images.githubusercontent.com/13754172/82246444-88f11100-98f9-11ea-83e7-ceec937ecfa8.png)

Reply:
By default when opening the repo though, I only see 
![image](https://user-images.githubusercontent.com/3580640/82246958-74614880-98fa-11ea-8fd9-188b1446eee4.png), not all the others.

Reply:
@keveleigh Yes, I think this Build Setting is specific to Examples Hub. Can you try opening ExamplesHub or ExamplesHubMainMenu?

Reply:
@keveleigh Oh, I see it is added through the Scene System profile, not by default.
I will revert the Build Settings file

Reply:
Reverted the Build Settings file